### DISCUSSION TITLE
IP banned against RecGov

### DISCUSSION THREAD

**Discussion by Schachte** [juftin/camply#272] [Q&A]
Curious if anyone is familiar with IP bans against RecGov? I'm failing out with continuous 403's and can confirm my VM IP is the only thing getting blocked by Cloudfront. 

<img width="1280" alt="image" src="https://github.com/juftin/camply/assets/7055226/1a4b8111-19e6-441e-9b34-261e74d2671d">

**Comment by Schachte** [comment]
This happened due to docker container being set to restart always, but `continuous` config param was false, which spams them continuously without a poll interval.

**Comment by Schachte** [comment]
Can confirm the IP ban was temporary

**Reply by juftin** [reply]
I've had this happen before when doing lots of rapid testing. @Schachte just curious, what rec area / campground were you searching?

**Reply by Schachte** [reply]
Yosemite. I just did the entire Yosemite rec area 2991. 

Btw, just scored a spot in Yosemite Valley I've been really wanting for next week thanks to you!

**Reply by juftin** [reply]
Oh interesting, I thought that might be the case. I believe Yosemite is actually one of the big culprits for IP bans from RecDotGov (moreso than other campgrounds / rec areas). They seem to have the highest bot traffic so `RecDotGov` is quick to place a temporary ban on your IP
